bin/install-deps.sh

```diff
@@ -16,8 +16,8 @@
 # Dependency Versions
 PROMETHEUS_VERSION=2.9.2
 GRAFANA_VERSION=6.7.4
-POSTGRES_EXPORTER_VERSION=0.7.0
-PGMONITOR_COMMIT='v3.2'
+POSTGRES_EXPORTER_VERSION=0.8.0
```

Comment:
With all exporter functionality now over on the Operator side, is this needed?  And therefore, do we still need to download the `postgres_exporter` a few lines down?

Reply:
No, that can be removed. Good catch, thanks!